app/controllers/application_controller.rb

```diff
@@ -1,2 +1,13 @@
 class ApplicationController < ActionController::Base
+  before_action :switch_locale
+
+  private
+  def switch_locale(&action)
```

Comment:
no block using in method, you can remove it
```suggestion
  def switch_locale
```